examples/custom-component/src/views/lite/components/Advanced.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+
+export class DisappearingText extends React.Component {
```

Comment:
They are mostly to showcase what's possible, but if there are genuinely useful components, i'd add them. A dropdown could be nice, but it would require adding a dependency to channel web, which is why I kept it as a custom one. Maybe an official channel web extension which includes more components? idk